cpp/src/IceStorm/Scanner.l

```diff
@@ -150,22 +162,22 @@ keyword [[:alpha:]]*
             s += static_cast<char>(c);
         }
     }
-    yylvalp->clear();
```

Comment:
This is the same variable, it's name was just changed is all.
`yylval` is the correct name for this variable.